src/mapstreeview.cpp

```diff
@@ -60,30 +64,30 @@ void MapsTreeView::_showContextMenu(const QPoint& point) {
     m_mapMenu->exec(viewport()->mapToGlobal(point));
 }
 
-void MapsTreeView::_disableActions() {
-    removeAction(m_newMapAction);
-    QObject::disconnect(m_newMapAction, SIGNAL(triggered()), this,
-                        SLOT(_onNewMapAction()));
-    QObject::disconnect(m_propertiesAction, SIGNAL(trigerred()), this,
-                        SLOT(_onPropertiesAction()));
```

Comment:
Qt disconnects and delete from parent in QObject destructors. We don't need to do it.